### ISSUE TITLE
Show Detail Popup on mouse hover

### ISSUE THREAD

**Issue by dupled** [nhn/tui.calendar#413]
## Version

1.12.5
## Development Environment

Chrome 77.0 | Windows 10
## Current Behavior

I'm unable to find the way to show the Detail Popup on mouse hover.

## Expected Behavior

The Detail Popup will show when hover on the schedule block.

**Comment by jungeun-cho** [comment]
@dupled 
The mouse hover event apis are not supported.

**Comment by gkoudjou** [comment]
Hi there!

Is this features planned in a future release ?

Thanks

**Comment by adhrinae** [comment]
@gkoudjou 

Not really. But you can leverage [the event delegation](https://github.com/nhn/tui.calendar/issues/267#issuecomment-1204623131) and [calling private method](https://github.com/nhn/tui.calendar/issues/555#issuecomment-1234928012) to do that.
